doc/md/integrations/github-actions.mdx

```diff
@@ -379,4 +380,33 @@ Configure this action with the following inputs:
 
 #### `cloud-token`
 
-**Required** The Atlas Cloud token to use for authentication.
\ No newline at end of file
+**Required** The Atlas Cloud token to use for authentication.
+
+## `ariga/setup-atlas`
+
+The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install Atlas from a github action.
+
+This is useful if your team wants to use Atlas via scripts or when you want to use features that are not exposed by other actions.
+
+### Example
+
+```yaml
+name: Deploy schema changes
+run-name: I'm using Atlas 🚀
+on: [push]
+jobs:
+  use-atlas:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ariga/setup-atlas@master
+      - run: atlas version
+      # - run: atlas schema apply ... 
+```
+
+### Reference
+
+The action supports the `Version` input:
+
+#### `Version`
```

Comment:
maybe give an example of the semantic versioning, such as:
`
with:
    Version: v0.13.2
`